lib/shared/bucketing-assembly-script/__tests__/types/configBody.test.ts

```diff
@@ -1,5 +1,10 @@
 import testData from '../../../bucketing-test-data/json-data/testData.json'
-import { testConfigBodyClass } from '../bucketingImportHelper'
+import {
+    testConfigBodyClass,
+    setConfigData,
```

Comment:
Should this be removed then?

Reply:
no that's still used by all the existing tests. Just added a new test for the new methods.